packages/block-editor/src/layouts/flow.js

```diff
@@ -67,22 +67,13 @@ export default {
 				info: alignmentInfo[ alignment ],
 			} ) );
 		}
-		const { contentSize, wideSize } = layout;
```

Comment:
Just double-checking — is the idea that this can be removed now because of the layout object being passed down? From testing, the issue of the wide width post template showing up using content size appears to have returned in this PR. I think the flow layout might still need the logic here (or elsewhere), so that the `alignwide` and `alignfull` classnames can be injected.

With TwentyTwentyTwo theme applied, it looks like the post template is being rendered as wide width on the site frontend, but content width in the site editor:

| Editor (note 650px) | Site frontend (note 1000px) |
| --- | --- |
| <img width="1796" alt="image" src="https://user-images.githubusercontent.com/14988353/215638319-b28ee15c-b0ca-4cce-9d84-8d340fc3377e.png"> | <img width="1257" alt="image" src="https://user-images.githubusercontent.com/14988353/215638519-c193bd2b-d670-4169-af4d-962b08dd66e4.png"> |

It looks like on the site frontend the post template block's output has both `is-layout-flow` and `alignwide` classes as expected, however in the site editor the `alignwide` class is not present.

Reply:
That code needs to be removed because it's causing align wide and full controls to _always_ show, even if the parent block doesn't have a content width. The problem is that the layout we're extracting content and wide size from there is the full theme layout, not the block one, so it always includes those properties if they're set by the theme. 

The specific problem those lines were fixing should now be fixed by the Query block deprecation. I only checked with the test markup from your PR, but will have a look at TT2 now.